src/app/tracing/dialog/station-properties/station-properties.component.ts

```diff
@@ -149,6 +149,10 @@ export class StationPropertiesComponent implements OnInit, OnDestroy {
     notListedProps: string[] = ['name', 'incoming', 'outgoing'];
     otherProperties: string[] = [];
 
```

Comment:
Please replace name `specialProps` by `conditionalNotListedPropsMap`.
and 
`station.isMeta === true` by 'station.isMeta === false`

Reply:
done!